gcm/message.go

```diff
@@ -34,6 +35,10 @@ func (m *Message) validate() error {
 		return fmt.Errorf("the message's RegistrationIDs field must not be nil")
 	}
 
+	if string(m.Priority) != "" && m.Priority != "normal" && m.Priority != "high" {
```

Comment:
Why do you need to cast? `m.Priority` is a string type. `m.Priority` is assigned `""` as a zero value.